content/docs/connectors/google.md

```diff
@@ -68,3 +73,14 @@ To get group fetching set up:
 3. Add the `serviceAccountFilePath` and `domainToAdminEmail` configuration options to your Dex config.
   - `serviceAccountFilePath` should point to the location of the service account JSON key file
   - `domainToAdminEmail` should map the base domain to the email address of a Google Workspace user with a minimum of the `Groups Reader (BETA)` Role assigned. The service account you created earlier will impersonate this user when making calls to the admin API. A valid user should be able to retrieve a list of groups when [testing the API](https://developers.google.com/admin-sdk/directory/v1/reference/groups/list#try-it).
+
+## Consent Screen
+
+Dex will set `prompt=consent` by default when redirecting users to Google
+authorization endpoint. This will force users to see the consent screen every
+time they log in.
+
+To change this behavior, you can set the `promptType` field in config file to
+any of the OIDC supported value. To skip the consent screen for every
```

Comment:
Grammar note, this should either read "to any OIDC-supported value" or "to any of the OIDC-supported values" (though I'm not sure what other supported values exist).